sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -165,13 +170,20 @@ private static void DumpActualDiagnostics(IEnumerable<Diagnostic> diagnostics)
 
         private static void VerifyBuildErrors(ImmutableArray<Diagnostic> diagnostics, Compilation compilation)
         {
-            var buildErrors = diagnostics.Where(d => d.Id.StartsWith("CS") && d.Severity == DiagnosticSeverity.Error);
+            //            var buildErrors = diagnostics.Where(d => d.Id.StartsWith("CS") && d.Severity == DiagnosticSeverity.Error);
+            var buildErrors = GetBuildErrors(diagnostics);
+
             var expectedBuildErrors = new IssueLocationCollector()
                 .GetExpectedBuildErrors(compilation.SyntaxTrees.Skip(1).FirstOrDefault()?.GetText().Lines)
                 .ToList();
             CompareActualToExpected(buildErrors, expectedBuildErrors, true);
         }
 
+        private static IEnumerable<Diagnostic> GetBuildErrors(IEnumerable<Diagnostic> diagnostics) =>
+            diagnostics.Where(d => d.Severity == DiagnosticSeverity.Error
+                && (d.Id.StartsWith("CS") || d.Id.StartsWith("BC"))
+                && !BuildErrorsToIgnore.Contains(d.Id));
+
```

Comment:
The PR could be checked in if the check for VB errors ("BC") was temporarily disabled again.